scripts/experimental/install_dev_osgeo.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+set -e
+
+## Install PROJ, GDAL, GEOS from source.
+##
+## 'latest' means installing the latest release version.
+
+## build ARGs
+NCPUS=${NCPUS:-"-1"}
+
+PROJ_VERSION=${PROJ_VERSION:-"latest"}
+GDAL_VERSION=${GDAL_VERSION:-"latest"}
+GEOS_VERSION=${GEOS_VERSION:-"latest"}
+
+CRAN=${CRAN_SOURCE:-"https://cloud.r-project.org"}
+echo "options(repos = c(CRAN = '${CRAN}'))" >>"${R_HOME}/etc/Rprofile.site"
+
+# a function to install apt packages only if they are not installed
+function apt_install() {
+    if ! dpkg -s "$@" >/dev/null 2>&1; then
+        if [ "$(find /var/lib/apt/lists/* | wc -l)" = "0" ]; then
+            apt-get update
+        fi
+        apt-get install -y --no-install-recommends "$@"
+    fi
+}
+
+function url_latest_gh_released_asset() {
+    wget -qO- "https://api.github.com/repos/$1/releases/latest" | grep -oP "(?<=\"browser_download_url\":\s\")https.*\.tar.gz(?=\")"
+}
+
+export DEBIAN_FRONTEND=noninteractive
+apt_install \
+    gdb \
+    git \
+    libcairo2-dev \
```

Comment:
@eitsupi can you add in cmake here? Optional for the current release but will be needed for gdal going forward. Thanks @rsbivand .

Reply:
I think cmake is (was) already included. look at line 64.